packages/@aws-cdk/aws-lambda/lib/scalable-function-attribute.ts

```diff
@@ -0,0 +1,88 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+import { Construct, IConstruct, Token } from '@aws-cdk/core';
+
+/**
+ * Interface for scalable attributes
+ */
+export interface IScalableFunctionAttribute extends IConstruct {
+  /**
+   * Scale out or in to keep utilization at a given level. The utilization is tracked by the
+   * LambdaProvisionedConcurrencyUtilization metric, emitted by lambda. See:
+   * https://docs.aws.amazon.com/lambda/latest/dg/monitoring-metrics.html#monitoring-metrics-concurrency
+   *
+   * Allowed values: 0.1 - 0.9.
+   */
+  scaleOnUtilization(options: UtilizationScalingOptions): void;
+  /**
+   * Scale out or in based on schedule.
+   */
+  scaleOnSchedule(id: string, actions: appscaling.ScalingSchedule): void;
+}
+
+/**
+ * Options for enabling Lambda utilization tracking
+ */
+export interface UtilizationScalingOptions extends appscaling.BaseTargetTrackingProps {
+  /**
+   * Utilization target for the attribute. For example, .5 indicates that 50 percent of allocated provisioned concurrency is in use.
+   *
+   * Allowed values: 0.1 - 0.9.
+   */
+  readonly utilizationTarget: number;
+}
+
+/**
+ * A scalable lambda alias attribute
+ */
+export class ScalableFunctionAttribute extends appscaling.BaseScalableAttribute {
+  constructor(scope: Construct, id: string, props: ScalableFunctionAttributeProps){
+    super(scope, id, props);
+  }
+  /**
+   * Scale out or in to keep utilization at a given level. The utilization is tracked by the
+   * LambdaProvisionedConcurrencyUtilization metric, emitted by lambda. See:
+   * https://docs.aws.amazon.com/lambda/latest/dg/monitoring-metrics.html#monitoring-metrics-concurrency
+   *
+   * Allowed values: 0.1 - 0.9.
```

Comment:
Same here. Drop this in favour of the documentation on `utilizationTarget` property.

Reply:
I have removed it in all places you commented on, since I rename the file it does not show it.